test-runner-frontend/src/Components/IdCell.test.js

```diff
@@ -1,40 +1,84 @@
 import IdCell from './IdCell';
 import { render, fireEvent, waitFor, screen, getByRole } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
+import { BrowserRouter, Router } from 'react-router-dom';
+import { MemoryRouter } from 'react-router-dom';
```

Comment:
we can merge these into one import. also YESSSSSSS for updating and fixing tests

Reply:
I found out BrowserRouter is enough so I'll delete MemoryRouter and Router.